### PR TITLE
Database setup

### PR DESCRIPTION
**Pull request by Carshy**
In the current project, I used a relational database to create the initial data structure for a vet clinic database. I created a table to store animals' information, inserted some data into it, and query it. Attached herein are two screenshots of the query information for my database.

screenshott1
screenshott2

screenshott3

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
```
Comment (ShahierNashaat) [inline queries.sql]: - [ ] Kindly notice that the query should find all animals whose name ends in "mon".

```sql
SELECT * FROM animals WHERE name LIKE '%mon';
```

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+SELECT * FROM animals WHERE date_of_birth BETWEEN '#2017-05-12#' AND '#2018-11-15#';
```
Comment (ShahierNashaat) [inline queries.sql]: - [ ] Kindly notice that the query should list the name of all animals born between 2016 and 2019.

```sql
SELECT name FROM animals WHERE EXTRACT(year FROM date_of_birth) BETWEEN 2016 AND 2019;
```

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @Carshy, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Carshy** [comment]
Hello, changes made as requested by the previous reviewer. I have added a third screenshot named screenshott3 to showcase the changes made and how the queries run based on the CHANGES REQUESTED. @ShahierNashaat  thanks the review was very helpful

**Review by mikethreels — APPROVED** [review]
Hi Musoko, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- good job following the instructions from the previous CR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**